include/zephyr/drivers/fuel_gauge.h

```diff
@@ -156,9 +166,22 @@ struct fuel_gauge_get_property {
 		uint16_t sbs_at_rate_time_to_empty;
 		/** FUEL_GAUGE_SBS_ATRATE_OK */
 		bool sbs_at_rate_ok;
+		/** FUEL_GAUGE_MANUFACTURE_DATE */
+		uint16_t manufacturer_date;
+		/** FUEL_GAUGE_SERIAL_NUMBER */
+		uint16_t serial_number;
 	} value;
 };
 
+/** Block properties are separated due to size */
+struct fuel_gauge_get_block_property {
+	/** Battery fuel gauge property to get */
+	uint16_t property_type;
+
+	/** Negative error status set by callee e.g. -ENOTSUP for an unsupported property */
+	int status;
```

Comment:
I suppose since we're only invoking the function with a single property this field is redundant as made obvious by your implementation in the SBS driver.

However, keeping it here would make migrating to supporting more than a single property much easier. What do you think we ought to do?

Reply:
@protterc Friendly ping for your comment here.

Reply:
Now that I see your comment. SGTM!